File: 283-move-zeroes/move-zeroes.cpp
class Solution {
public:
    void moveZeroes(vector<int>& nums) {
        int m=0;
      for(int i=0;i<nums.size();i++){
if(nums[i]!=0){
    nums[m++]=nums[i];
}
      }

      for(int i=m;i<nums.size();i++){
        nums[i]=0;
      }
    }  
};

File: 283-move-zeroes/move-zeroes.py
class Solution:
    def moveZeroes(self, nums: List[int]) -> None:
        nonZero=0
        for i in range(len(nums)):
            if nums[i]!=0:
                nums[nonZero]=nums[i]
                nonZero+=1
        while nonZero<len(nums):
            nums[nonZero]=0
            nonZero+=1
        
        